conf/ocs_basic_install.yml

```diff
@@ -7,4 +7,4 @@ ENV_DATA:
   platform: 'aws'
   worker_replicas: 3  # here you can define diff number of workers
   master_replicas: 3
-  rook_image: "rook/ceph:v1.0.0-256.g01ba50e" # you can overwrite default rook image
+  rook_image: "rook/ceph:master" # you can overwrite default rook image
```

Comment:
Not sue, if I miss something, how the change of `rook_image` relates to this PR ("Environment checker improvement")?

Reply:
fixed